include/InstrumentPlayHandle.h

```diff
@@ -56,7 +56,7 @@ class EXPORT InstrumentPlayHandle : public PlayHandle
 		do
 		{
 			nphsLeft = false;
-			foreach( const NotePlayHandle * cnph, nphv )
+			for( const NotePlayHandle * cnph : nphv )
```

Comment:
I would rename `cnph` to something like `notePlayHandle` to make things more readable in the loop.